src/index.ts

```diff
@@ -293,7 +299,11 @@ async function statusOk(res: Response, logger: Logger): Promise<boolean> {
     return String(res.status)[0] === '2'
 }
 
-export function getProperties(event: PluginEvent, propertiesToInclude: string): Properties {
+function getNestedProperty(obj: any, path: string): any {
+    return path.split('.').reduce((acc, part) => acc && acc[part], obj);
+}
+
+export function getProperties(event: PluginEvent, propertiesToInclude: string, fieldMappings: FieldMappings = {}): Properties {
```

Comment:
you'll need to add/update the unit tests (i can help if needed) 

that'll have the added benefit of capturing some of what you've figured out for the future traveller too